bindings/python/Jamfile

```diff
@@ -25,6 +25,20 @@ if $(LIBTORRENT_PYTHON_INTERPRETER)
 	using python : : "$(LIBTORRENT_PYTHON_INTERPRETER)" : : : <libtorrent-python>on ;
 }
 
+local versions = [ feature.values python ] ;
+local version ;
+for local v in $(versions)
```

Comment:
this loop makes it look like you just pick the last element in `versions`, is that right?

Reply:
I'm not really familiar with Jamfiles, but I wanted to get Python version that we want to build. This is needed on macOS so we use library for correct Python version. Previously, it was always just for Python 3.7.

Reply:
with boost-build there isn't necessarily *one* build. There are build features and they are combined to form the targets. So, technically, a python version is associated with a *target*, not at this global level. I can look into making that change, I don't think it's that important.

Reply:
I fixed this in `RC_1_2` and more recent branches. It looks like your rebase overwrote it. I think the version on the left is preferred